mysql/cloudmysql/cloudmysql.go

```diff
@@ -79,6 +80,28 @@ func Open(ctx context.Context, certSource proxy.CertSource, params *Params) (*sq
 	return sql.OpenDB(connector(cfg.FormatDSN())), nil
 }
 
+// OpenGAE opens a Cloud SQL database on Google App Engine (GAE) using the
+// environment variables $DB_USER, $DB_DATABASE, $DB_INSTANCE, and
+// $DB_PASSWORD.
+func OpenGAE(ctx context.Context) (*sql.DB, error) {
```

Comment:
ctx is unused?

Reply:
True, although that's also true of the Open function above. Removing it there as well.